packages/sdk/src/router/v6/router.ts

```diff
@@ -141,6 +147,8 @@ export class Router {
       directFillingData?: any;
       // Wallet used for relaying the fill transaction
       relayer?: string;
+      permit?: any;
```

Comment:
Instead of taking the permit and the signature as parameters, we should be generating them in this method, like we do here:
https://github.com/reservoirprotocol/core/blob/d724f1f31e6332f85e9511114279f801eaf310a7/packages/sdk/src/router/v6/router.ts#L1657-L1724

And here:
https://github.com/reservoirprotocol/core/blob/d724f1f31e6332f85e9511114279f801eaf310a7/packages/sdk/src/router/v6/router.ts#L2161-L2176